src/components/header.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Header from './header';
+
```

Comment:
- [ ] As I suggested above that this would be great if you you should have snapshots for all React Components so it would be great if you make a snapshot of this header.js component 👍